lumigator/python/mzai/jobs/inference/inference.py

```diff
@@ -86,24 +86,24 @@ def run_inference(config: Box) -> Path:
 
     # Enable / disable tqdm
     input_samples = dataset["examples"]
-    dataset_iterable = tqdm(input_samples) if config.evaluation.enable_tqdm else input_samples
+    dataset_iterable = tqdm(input_samples) if config.job.enable_tqdm else input_samples
```

Comment:
have been thinking...do we still want the option to disable tqdm? we should probably leave on by default

Reply:
I always found it useful, but am not against leaving a chance to turn it off (one use case could be e.g. huge datasets which might generate a lot of ray logs per run)